packages/gasket-plugin-nextjs/generator/app/.babelrc

```diff
@@ -1,12 +1,16 @@
 {
-  "presets": [
```

Comment:
I tried this against the canary app. While tests work (`npm test`), builds (`npm run build`) break with it.

It looks like having `next/babel` without customization in the base preset definition works:
```
  "presets": [
    "next/babel"
  ],
  "env:" ...
```